src/CheckboxCatalog.js

```diff
@@ -4,7 +4,7 @@ import {MDCCheckbox} from '@material/checkbox/index';
 
 import './styles/CheckboxCatalog.scss';
 
-const CheckboxCatalog = () => {
+const CheckboxCatalog = (props) => {
```

Comment:
I assume changes to checkbox/radio/etc are for reuse within the snippet options? Did we try using MDC React components for those?

Reply:
We didn't since the components are relatively simple to replicate without any issues.